users/models.py

```diff
@@ -14,16 +14,20 @@ class User(AbstractUser):
     address_street = models.CharField(max_length=100, blank=True)
     address_home_number = models.CharField(max_length=10, blank=True)
     address_room_number = models.CharField(max_length=10, blank=True)
-    phone = models.CharField(max_length=18)
+    phone = models.CharField(max_length=18, blank=True)
     avatar = models.ImageField(
         upload_to='photos/profiles',
         verbose_name='User\'s photo',
         blank=True)
     zip_code = models.CharField(max_length=12, blank=True)
+    email_confirmed = models.BooleanField(default=False)
+    username = models.CharField(max_length=150, verbose_name='username')
```

Comment:
Not sure I understand the meaning of this field. Why do we need it?

Reply:
@dkrukouski I think the idea is to make the field not unique. But question to @Zomba4ok: why didn't you do the field nullable?